ethereum/core/src/test/java/org/hyperledger/besu/ethereum/core/encoding/TransactionRLPEncoderTest.java

```diff
@@ -66,17 +65,4 @@ public void encodeEIP1559TxNominalCase() {
         .isEqualTo(output.encoded().toHexString());
     ExperimentalEIPs.eip1559Enabled = ExperimentalEIPs.EIP1559_ENABLED_DEFAULT_VALUE;
   }
-
-  @Test
-  public void encodeEIP1559TxFailureNotEnabled() {
-    ExperimentalEIPs.eip1559Enabled = false;
-    assertThatThrownBy(
-            () ->
-                TransactionRLPEncoder.encode(
-                    TransactionRLPDecoder.decode(RLP.input(Bytes.fromHexString(EIP1559_TX_RLP))),
-                    new BytesValueRLPOutput()))
-        .isInstanceOf(RuntimeException.class)
-        .hasMessageContaining("EIP-1559 feature flag");
-    ExperimentalEIPs.eip1559Enabled = ExperimentalEIPs.EIP1559_ENABLED_DEFAULT_VALUE;
-  }
```

Comment:
We don't fail at deseriailization anymore. We always deserialize and reject in transaction validation based on what fork we're on